code/__DEFINES/scenarios.dm

```diff
@@ -0,0 +1,4 @@
+/// This mission is canon.
+#define SCENARIO_TYPE_CANON				1
+/// This mission is not canon.
+#define SCENARIO_TYPE_NONCANON				2
```

Comment:
Probably would be better to be strings instead of numbers, since it doesn't seem we do anything number-y with it and stack traces / vv and the likes wouldn't be able to show the define name?